src/web/editors/digitaljs.ts

```diff
@@ -45,6 +46,21 @@ export class DigitalJSEditor extends BaseEditor {
                 document: document.getText()
             });
         }
+        else if (message.type === 'requestSave') {
+            // TODO: figure out better way to save relative to project root
```

Comment:
We can probably attempt to match the path of the `.digitaljs.json` file to a project root (`Project.root` in `src/projects/project.ts`). If one matches, we can use it as default, otherwise we can use the first workspace folder.

Reply:
The approach I suggested is way too complicated. We can just use the parent directory of the `.digitaljs.json` file as the default.

Reply:
I took a slightly different approach: it will look upward from the `digitaljs.json` file until it's found a workspace directory. That should also account for the cases where the json is not in the project root. It does still default to the file's parent directory in case it can't find the project root however.